plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/internal/common/core/util/CommandLocationLookupStrategy.java

```diff
@@ -29,12 +29,22 @@
 	private static final String LINUX_PATHVAR = "PATH";
 	private static final String WINDOWS_PATHVAR = "Path";
 
+	private static final String LINUX_SEPARATOR = ":";
+	private static final String WIN_SEPARATOR = ";";	
+	
 	public static final CommandLocationLookupStrategy WINDOWS_STRATEGY = 
-			new CommandLocationLookupStrategy(WINDOWS_WHERE, ";", WINDOWS_PATHVAR, new String[]{".exe", ".com"}, null);
+			new CommandLocationLookupStrategy(WINDOWS_WHERE, 
+					// Windows can use either separator or path variable
+					// based on whether eclipse was launched from cygwin or not
+					System.getenv().get(WINDOWS_PATHVAR) == null ? LINUX_SEPARATOR : WIN_SEPARATOR,
```

Comment:
This doesn't seem right, Denis.  You sent me this on hipchat last week:

and path is
/c/Users/eskimo/bin:/mingw64/bin:/usr/local/bin:/usr/bin:/bin:/mingw64/bin:/usr/bi
n:/c/Users/eskimo/bin:/c/ProgramData/Oracle/Java/javapath:/c/Programs/Python27:/c/Programs/Python27/Scripts:/c/Windows/system32:/c/Windows:/c/Windows/System32/Wbem:/c/Windows/System32/WindowsPowerShell/v1.0:/c/Programs/nvm:/c/Programs/node:/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common:/c/Program Files (x86)/Skype/Phone:/c/Users/eskimo/.dnx/bin:/c/Program Files/Microsoft DNX/Dnvm:/c/Program Files (x86)/Windows Kits/8.1/Windows Performance Toolkit:/c/Program Files/Puppet Labs/Puppet/bin:/c/Programs/Ruby193/bin:/c/DeveloperPlatform/cdk/bin:/c/DeveloperPlatform/vagrant/bin:/c/DeveloperPlatform/ssh-rsync/bin:/c/Windows/system32:/c/Windows:/c/Windows/System32/Wbem:/c/Windows/System32/WindowsPowerShell/v1.0:/c/Programs/nvm:/c/Programs/node:/c/Program Files (x86)/WiX Toolset v3.10/bin:/c/Users/eskimo/AppData/Local/atom/bin:/c/Programs/7zip-extra:/usr/bin/vendor_perl:/usr/bin/core_perl

You can see it's using :  and not ;